docs/user/advanced.rst

```diff
@@ -269,6 +269,28 @@ If you specify a wrong path or an invalid cert, you'll get a SSLError::
 .. warning:: The private key to your local certificate *must* be unencrypted.
    Currently, Requests does not support using encrypted keys.
 
+.. _sni-support:
+
+SNI support with legacy Python
+------------------------------
+
+.. versionadded:: 2.4.1
```

Comment:
I've put this `versionadded` directive, but since it's quoting a very old version, it might not make sense?

Reply:
This was originally added for SNI support but further is really just a way of providing modern OpenSSL for old versions of Python too. It allows them to communicate with modern websites regardless of SNI.

Reply:
Thanks for your feedback. I've updated the change to be less focused on SNI, and direct to the upstream documentation for `urllib3`.